samples/snippets/src/main/java/com/example/spanner/CreateDatabaseWithVersionRetentionPeriodSample.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+// [START spanner_create_database_with_version_retention_period]
```

Comment:
Typically, the start region comes just before the method definition. In this case, it should be line 33, so users don't have to read all the import statements.

Reply:
The formatting guideline states that `import` statements should be included within the region tags: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#class-structure

Reply:
Thanks Knut. Did this guidance change recently? If so, I'm curious what the rationale was. If not, why are so many of the existing samples different?

Reply:
Not (very) recently. I think it has been like this for at least the last 7-8 months, and maybe also longer. But there are a lot of older samples in the Spanner repository that were created before this styleguide was available (or enforced), so I think that that is the reason that there are many samples in this repository that don't adhere to the guide. I don't know whether that is also the case for other repositories, though.